.gitlab-ci.yml

```diff
@@ -243,6 +243,24 @@ trigger_build_kernels:
       - local: .gitlab/kernel_version_testing.yml
     strategy: depend
 
+trigger_tests:
+  stage: test
+  except: [tags, schedules]
+  variables:
+    RUN_KITCHEN_TESTS: "false"
```

Comment:
Shouldn't we run all kitchen tests for this kind of changes as well? It seems similar in term of risks to changes in omnibus-software

Reply:
previous tests were only testing to build the agent, I'll rather stick to it to not increase to much CI time